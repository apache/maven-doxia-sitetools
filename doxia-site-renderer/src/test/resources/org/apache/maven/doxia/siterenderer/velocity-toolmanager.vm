$esc.html('Maven 3 & 4')
$display.truncate("This is a long string.", 10)

We have the following keys in the context:
#foreach( $key in $sorter.sort( $context.keys ) )
  $display.printf( "%2s", ${foreach.count} ). $key
#end
