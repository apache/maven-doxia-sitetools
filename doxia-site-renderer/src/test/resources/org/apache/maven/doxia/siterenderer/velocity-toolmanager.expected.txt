Maven 3 &amp; 4
This is...

We have the following keys in the context:
   1. PathTool
   2. StringUtils
   3. alignedFileName
   4. alternator
   5. authors
   6. bodyContent
   7. class
   8. context
   9. convert
  10. currentFileName
  11. date
  12. decoration
  13. display
  14. docRenderingContext
  15. documentDate
  16. doxiaSiteRendererVersion
  17. esc
  18. field
  19. headContent
  20. link
  21. locale
  22. loop
  23. math
  24. number
  25. plexus
  26. publishDate
  27. relativePath
  28. render
  29. shortTitle
  30. sorter
  31. supportedLocales
  32. text
  33. title
  34. xml
